.pre-commit-config.yaml

```diff
@@ -8,12 +8,11 @@ repos:
   rev: 6.0.0
   hooks:
   - id: flake8
-- repo: https://github.com/PyCQA/pylint
-  rev: v2.15.6
-  hooks:
-  - id: pylint
-    args: [conftest.py, docs, extras, scrapy, setup.py, tests]
```

Comment:
Done to address https://github.com/scrapy/scrapy/issues/5810, since otherwise we would be removing pylint from checks.yml in this pull request to re-add it when we remove pylint from pre-commit.